content/en/contribute/medic/product-development-process/cadence-of-events.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "High-level Cadence of Events"
+linkTitle: "High-level Cadence of Events"
+weight: 2
+description: >
+    High-level Cadence of Events of the Product Team
```

Comment:
We should update with a more descriptive sentence that acts like a mini preview. For instance: 
```suggestion
    Schedule of activities for Focused Working Groups
```

This suggestion should change if the page title becomes "Schedule of activities" :)